gradle.properties

```diff
@@ -14,4 +14,8 @@ POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENCE_DIST=repo
 
 POM_DEVELOPER_ID=vanniktech
-POM_DEVELOPER_NAME=Niklas Baudy
\ No newline at end of file
+POM_DEVELOPER_NAME=Niklas Baudy
+
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
Can't we set this to false?

Reply:
No. I mentioned it somewhere earlier that the library we use for ViewPager animations still didn't migrate to AndroidX so we need this.

Reply:
Ah right!

Reply:
Sorry, I forgot

Reply:
No worries! :)